dask/array/tests/test_random.py

```diff
@@ -7,55 +7,73 @@
 import dask
 import dask.array as da
 from dask.array.core import Array
-from dask.array.random import exponential, normal, random
 from dask.array.utils import assert_eq
 from dask.multiprocessing import _dumps, _loads
 from dask.utils import key_split
 
 
-def test_RandomState():
-    state = da.random.RandomState(5)
-    x = state.normal(10, 1, size=10, chunks=5)
+@pytest.fixture(params=[da.random.RandomState, da.random.default_rng])
+def generator_class(request):
+    return request.param
+
+
+def test_generators(generator_class):
+    state = generator_class(5)
+    x = state.normal(10, 1, size=10, chunks=5).compute()
```

Comment:
Why do we need to `compute()` results here? `assert_eq` should take care of that, plus computing here will potentially lose things attributes such as `meta`.

Reply:
fixed with b7c15b3